libraries/net/node_impl.hxx

```diff
@@ -573,12 +585,13 @@ class node_impl : public peer_connection_delegate
       void listen_to_p2p_network();
       void connect_to_p2p_network();
       void add_node( const fc::ip::endpoint& ep );
-      void set_advertise_algorithm( std::string algo, const fc::optional<std::vector<std::string>>& advertise_list );
+      void set_advertise_algorithm( std::string algo, const fc::optional<std::vector<std::string>>& advertise_or_exclude_list );
       void add_seed_node( const std::string& seed_string );
       void initiate_connect_to(const peer_connection_ptr& peer);
       void connect_to_endpoint(const fc::ip::endpoint& ep);
       void listen_on_endpoint(const fc::ip::endpoint& ep , bool wait_if_not_available);
```

Comment:
I think it's better to rename this function and the following functions for better readability. These functions are for changing configurations, but the names look like for doing something else.
* `listen_on_endpoint` -> `set_listen_endpoint`
* `accept_incoming_connections` -> `set_accept_incoming_connections`
* `connect_to_new_peers` -> `set_connect_to_new_peers`
* `listen_on_port` -> `set_listen_port`

Reply:
Agreed. Will do.